apps/src/templates/census2017/CensusForm.jsx

```diff
@@ -0,0 +1,313 @@
+                  {i18n.censusFollowUpTellUsMore()}
+                </div>
+                <textarea
+                  type="text"
+                  name="followUpMore_s"
+                  value={this.state.submission.followUpMore}
+                  onChange={this.handleChange.bind(this, 'followUpMore')}
+                  style={styles.textArea}
+                />
+              </label>
+            </div>
+          )}
+          <label>
+            <div style={styles.question}>
+              {i18n.censusConnection()}
+            </div>
+            <select
+              name="role_s"
+              value={this.state.submission.role}
+              onChange={this.handleChange.bind(this, 'role')}
+              style={styles.option}
+            >
+              {roleOptions.map((role, index) =>
+                <option
+                  value={role}
+                  key={index}
+                >
+                  {role}
+                </option>
+              )}
+            </select>
+          </label>
+          <div style={styles.personalQuestionsBox}>
+            <div style={styles.personalQuestion}>
+              <label>
+                <div style={styles.question}>
+                  {i18n.yourName()}
+                </div>
+                <input
+                  type="text"
+                  name="name_s"
+                  value={this.state.submission.name}
+                  onChange={this.handleChange.bind(this, 'name')}
+                  placeholder={i18n.yourName()}
+                  style={styles.input}
+                />
+              </label>
+            </div>
+            <div style={styles.personalQuestion}>
+              <label>
+                <div style={styles.question}>
+                  {i18n.yourEmail()}
+                </div>
+                <input
+                  type="text"
+                  name="email_s"
+                  value={this.state.submission.email}
+                  onChange={this.handleChange.bind(this, 'email')}
+                  placeholder={i18n.yourEmailPlaceholder()}
+                  style={styles.input}
+                />
+              </label>
+            </div>
+          </div>
+          <div style={styles.pledgeBox}>
+            <label>
+              <input
+                type="checkbox"
+                name="pledge_b"
+                checked={selected.includes("pledge_b")}
+                onChange={() => this.toggle("pledge_b")}
+                style={styles.checkbox}
+              />
+              <span style={styles.pledge}>
+                {pledge}
+              </span>
+            </label>
+          </div>
+          <Button
+            id="submit-button"
+            onClick={() => this.censusFormSubmit()}
+            color={Button.ButtonColor.orange}
+            text={i18n.submit()}
+            size={Button.ButtonSize.large}
+          />
+        </form>
+        <div
+          id="thank-you"
```

Comment:
For here and `census-form` if we use state instead of jquery to determine whether or not they are hidden, that will probably also allow us to get rid of the ids.